docs/src/start/getting-started.md

```diff
@@ -512,11 +512,11 @@ To see the content of the log files in Cumulocity IoT, take the following steps:
 
 ![Request Log file](./images/RequestLogfile.png)
 
-If `c8y-log-plugin.toml` is added to the `c8y-configuration-plugin.toml` it is possible to do the administration from there.
+If `tedge-log-plugin.toml` is added to the `c8y-configuration-plugin.toml` it is possible to do the administration from there.
 
-<!-- TODO: Check if restarting the c8y-log-plugin is required each time the log file is edited -->
+<!-- TODO: Check if restarting the tedge-log-plugin is required each time the log file is edited -->
```

Comment:
That line can be removed as the log plugin had already supported dynamic config file updates using file watcher.

Reply:
I'm going to double check that, because I was pretty sure this is actually not working.